internal/pkg/plugin/jiragithub/create.go

```diff
@@ -0,0 +1,36 @@
+package jiragithub
+
+import (
+	ga "github.com/merico-dev/stream/internal/pkg/plugin/githubactions"
+	"github.com/merico-dev/stream/pkg/util/github"
+)
+
+// Create sets up jira-github-integ workflows.
+func Create(options map[string]interface{}) (map[string]interface{}, error) {
+	opt, err := parseAndValidateOptions(options)
+	if err != nil {
+		return nil, err
+	}
+
+	ghOptions := &github.Option{
+		Owner:    opt.Owner,
+		Repo:     opt.Repo,
+		NeedAuth: true,
+	}
+	gitHubClient, err := github.NewClient(ghOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, w := range workflows {
+		if err := gitHubClient.AddWorkflow(w, opt.Branch); err != nil {
+			return nil, err
+		}
+	}
+
+	if err := setRepoSecrets(gitHubClient); err != nil {
+		return nil, err
+	}
+
+	return ga.BuildState(opt.Owner, opt.Repo), nil
```

Comment:
In this case, `jira-github-integ` and `githubactions` plugins are both maintained by us, but in theory, DevStream plugins will be created and maintained by different committers.

So, it's customary that plugin A should not depend on plugin B's code.

Here, "BuildState" actually is part of the githubactions' internal (private) code.

If similarities are found between different plugins, we could theoretically create a public module under `/pkg`.

In this specific situation, I don't think this is the right think to do, though, because it's building "state" that specifically belongs to one plugin. Although the state are exactly the same for these two plugins for now, they might differ in the future.